src/Components/Guestbook/Guestbook.jsx

```diff
@@ -51,11 +59,8 @@ export default function Guestbook() {
 
                 {user && (
                     <button
-                        type='button' onClick={() => {
-                            setUser('')
-                            setName('')
-                        }}>
-                        Not, {user} ?
+                        type='button' onClick={handleSignout}>
+                        Not, {user}?, Click to Sign out!
```

Comment:
With swapping to using the Auth context and username from that, you won't actually have a field for the user to enter their name directly in anymore. It should be automatically used from context based on the login values. 

This means that you'll need to update the function `updateUserName()` to reference the `user` value that you're getting from context instead of the `name` value that you had previously from the `userNameInput`. In the end, you won't need the `userNameInput` function at all anymore since you won't be editing it yourself anymore. 

Let me know if this isn't clear and I can help clarify.